media/js/privacy/privacy-quantum-firefox.js

```diff
@@ -0,0 +1,61 @@
+(function() {
+    'use strict';
+
+    var strings = document.getElementById('strings');
+
+    /**
+     * Check for feature support
+     */
+    function supportsBaselineJS() {
+        return 'querySelectorAll' in document &&
+               'addEventListener' in window;
+    }
+
+    /**
+     * Adds the data choices widget(Firefox only) to the first sub-section under
+     * "Firefox by default shares data to".
+     * @param {Object} section - The section to which the widget
+     * will be added
+     */
+    function addDataChoicesWidget(section) {
+        var container = document.createElement('div');
+        var copyContainer = document.createElement('p');
+        var button = document.createElement('button');
+
+        container.setAttribute('class', 'data-choices');
+
+        copyContainer.textContent = strings.dataset.choicesCopy;
+
+        button.textContent = strings.dataset.choicesButton;
+        button.setAttribute('id', 'choose');
+        button.setAttribute('type', 'button');
+
+        container.appendChild(copyContainer);
+        container.appendChild(button);
+
+        section.appendChild(container);
+
+        // handle clicks on the data choices "Choose" button
+        $('#choose').on('click', function() {
+            // if the uitour did not load, just return
+            if (Mozilla.UITour === undefined) {
+                return;
+            }
+
+            Mozilla.UITour.openPreferences('privacy-reports');
+        });
+    }
+
+    // Don't execute if features aren't supported and client isn't desktop Firefox
+    if (supportsBaselineJS() && Mozilla.Client.isFirefoxDesktop) {
+        var topicHeaders = document.querySelectorAll('.privacy-body .content-girdle > section');
```

Comment:
If these are to be considered page scoped variables, it's good practise to leave the declarations at the top of the file for readability. So at the top of the file just`var topicHeaders;`. Then do the assignments here e.g. `topicHeaders = document.querySelectorAll('.privacy-body .content-girdle > section');`

You also still need to do this for the other file.

I do see the pill-shaped button now (probably my bad last time).